packages/spacecat-shared-rum-api-client/src/functions/opportunities/high-organic-low-ctr.js

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright 2024 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+
+import trafficAcquisition from '../traffic-acquisition.js';
+import { getCTRByUrl, getSiteAvgCTR } from '../../common/aggregateFns.js';
+
+const DAILY_EARNED_THRESHOLD = 5000;
+
+function convertToOpportunity(traffic) {
+  const { url, total, ctr } = traffic;
+
+  return {
+    type: 'high-organic-low-ctr',
+    page: url,
+    screenshot: '',
+    trackedPageKPIName: 'Click Through Rate',
+    trackedPageKPIValue: ctr,
+    pageViews: total,
+    samples: total, // todo: get the actual number of samples
+    metrics: [], // none needed
```

Comment:
may be its better to add the siteAverageCTR and page CTR in the metrics

Reply:
done